worlds/tits_the_3rd/regions.py

```diff
@@ -71,6 +83,110 @@ def connect_regions(multiworld: MultiWorld, player: int):
         multiworld, player, RegionName.jade_corridor_arseille, RegionName.jade_corridor_expansion_area_1, lambda state: state.has(ItemName.jade_corridor_arseille_unlock, player, 1)
     )
 
+    connect_region(
+        multiworld,
+        player,
+        RegionName.jade_corridor_expansion_area_2,
+        RegionName.night_grancel_south,
+    )
+    connect_region(
+        multiworld,
+        player,
+        RegionName.night_grancel_south,
+        RegionName.jade_corridor_expansion_area_2,
+    )
+
```

Comment:
Inconsistent spacing, should all be no space or single space between all of them

Reply:
I think in this case its fine, there's no spaces for bidrectional connections.